custom_components/coway/sensor.py

```diff
@@ -34,12 +36,14 @@ async def async_setup_entry(
     sensors = []
 
     for purifier_id, purifier_data in coordinator.data.purifiers.items():
+            _LOGGER.debug(f'PURIFIER DATA for {purifier_id}: {purifier_data}\n')
             if purifier_data.mcu_version is None:
                 sensors.append(AirQualityIndex(coordinator, purifier_id))
             sensors.extend((
                 PreFilter(coordinator, purifier_id),
                 MAX2Filter(coordinator, purifier_id),
                 ParticulateMatter10(coordinator, purifier_id),
+                ParticulateMatter25(coordinator, purifier_id),
```

Comment:
The ParticulateMatter25 sensor was removed a while ago since 300s and 400s purifiers always report 0 making the sensor useless. With the way things are here, the sensor will once again be recreated for 300S and 400S models. What needs to be done is have the sensor only be added if the purifier model is an IconS.